pythonx/ncm2_pyclang_proc.py

```diff
@@ -69,7 +69,7 @@ def worker_loop(self):
             except Exception as ex:
                 from neovim import Nvim
                 nvim = self.nvim  # type: Nvim
-                def raise_ex():
+                class raise_ex():
```

Comment:
But if don't change to `class`, it can't pop up an accurate error message.  Only display `TypeError: exceptions must derive from BaseException`
Is there any better way to handle this?